src/mr.roboto/src/mr/roboto/subscriber.py

```diff
@@ -272,6 +276,11 @@ def run(self):
                 f"Learn about the Plone Contributor Agreement: {self.cla_url} "
                 f"How to add more emails to your GitHub account: "
                 f"{self.github_help_setup_email_url} "
+                "\n\n"
+                "If you have sent in your Plone Contributor Agreement, "
+                "and received an invitation to join the Plone GitHub organization, "
+                "but your emails are not known, then you might need to add "
```

Comment:
Remove redundant clause. I was in a rush.

```suggestion
                "then you might need to add "
```